src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerEventListener.kt

```diff
@@ -40,8 +40,7 @@ class PrisonerEventListener(
 
         "OFFENDER-INSERTED", "OFFENDER-UPDATED", "OFFENDER_DETAILS-CHANGED",
         "OFFENDER_ALIAS-CHANGED", "OFFENDER_PHYSICAL_DETAILS-CHANGED",
-        "OFFENDER_IDENTIFIER-UPDATED",
-        "ASSESSMENT-UPDATED",
+        "OFFENDER_IDENTIFIER-UPDATED", "ASSESSMENT-UPDATED", "OFFENDER_CHARGES-UPDATED",
```

Comment:
Is this already in the filter in the cloud platform repo?

Reply:
doing it now - https://github.com/ministryofjustice/cloud-platform-environments/pull/16508